### DISCUSSION TITLE
1.5.3 Release of Kade Engine

### DISCUSSION THREAD

**Discussion by Kade-github** [KadeArchive/Kade-Engine#924] [General]
View all of the changes [here](https://kadedev.github.io/Kade-Engine/changelogs/changelog-1.5.3)

![image](https://user-images.githubusercontent.com/26305836/122657367-c7b65380-d117-11eb-8e4b-4ee451986ee1.png)

Mirror: https://drive.google.com/file/d/1Y_QSqNxbNyl7EyritFe6KKzaO0vPiZEw/view?usp=sharing

<hr /><em>This discussion was created from the release <a href='https://github.com/KadeDev/Kade-Engine/releases/tag/1.5.3'>1.5.3 Release of Kade Engine</a>.</em>

**Comment by notsouguy** [comment]
is there a way to disable it

**Reply by Kade-github** [reply]
disable what

**Reply by juney2008** [reply]
the results screen is what hes saying.

**Reply by Kade-github** [reply]
there's a option for it in misc

**Reply by stub-official-band** [reply]
anyways... this is amazing

**Reply by stub-official-band** [reply]
I'm so glad flamingo made a vid on this

**Comment by TheALt-creator** [comment]
will there be a window build of 1.5.3? since I'm not good a source codes.

**Comment by BombasticTom** [comment]
this update is sick!

**Comment by BombasticTom** [comment]
Was the week cleared section supposed to show the full week? It only shows me score for the last song and not the max score
![image](https://user-images.githubusercontent.com/85838311/122677047-d61c7200-d1e0-11eb-88b8-e6d4f3dd9ce7.png)

**Reply by BombasticTom** [reply]
Also the view replay button in the stage cleared state is broken

**Comment by LonelyMusicDisc** [comment]
shame there's no linux or mac build, hopefully the next one

**Reply by isakube** [reply]
build the game lol

**Reply by TentaRJ** [reply]
> build the game lol

actually true

**Reply by LonelyMusicDisc** [reply]
> build the game lol

I would but I can't rebuild extension-webm on linux for some reason :/

**Reply by puyoxyz** [reply]
kade likes to break linux and mac i'll fix it at some point when i stop being lazy

**Reply by hazelskaito** [reply]
> build the game lol

idk how to i need tut for mac

**Reply by FChesco** [reply]
> build the game lol

yea

**Comment by Yayayayayaaaaaaaaaaaaaaa** [comment]
im sorry but how do you get this i downloaded this and it put no option to start the game. can someone help please?

**Comment by TheHENOOB** [comment]
Interesting concept, but i think instead of using the default flixel text it could use hand drawn assets like the main menu

**Reply by isakube** [reply]
it fits with the vibe doe

**Reply by TheHENOOB** [reply]
If that's the case, sure it looks nice, but like using a custom font or making the "Song Clear!" text a custom sprite for it, i think it will look a little better

**Comment by PurSnake** [comment]
source/PlayState.hx:3264: characters 17-32 : Static access to instance field SKIP_STEP_LIMIT is not allowed
H:/Haxe/haxe/lib/polymod/git/polymod/format/XMLMerge.hx:189: characters 10-23 : Warning : This typedef is deprecated in favor of haxe.xml.Access
H:/Haxe/haxe/lib/polymod/git/polymod/format/XMLMerge.hx:189: characters 30-43 : Warning : This typedef is deprecated in favor of haxe.xml.Access

What can I do with this? Just download

**Comment by DOCTORDEADHEAD** [comment]
Any plans on an option to move the REPLAY text from the middle of the screen? I'm tryna grab screenshots of a mod and it feels a bit intrusive; It could instead be on the bottom or something.

**Comment by Megaman9009** [comment]
Is there anyway to update it without porting everything? I want to update my mod but I dont feel like porting all my assets to the new source code. Any way for it to just update my existing source code

**Comment by speebyboug** [comment]
Every time I try to download it, it downloads for a few minutes, then stops and says "Failed - Forbidden."

**Comment by DragoElMago** [comment]
yo i got a problem when charting, when i put a note i can't take it off unless i select "swap section" and is bothering me tbh because it works really weird (https://youtu.be/D9NnlKz6NZ0)

**Comment by UltraStarsShit** [comment]
why it keep error when i save the notes i make and i put it in the data and i rename it. i rename in freeplaylist too can you ask why error

**Comment by CubicPlays** [comment]
Why is it keep saying
"source/PlayState.hx:3264: characters 17-32 : Cannot access private field SKIP_STEP_LIMIT
source/PlayState.hx:3264: characters 17-32 : Cannot access private field SKIP_STEP_LIMIT"
idk why please help

**Comment by ItzEazich001** [comment]
How do I remove the Kade Engine update window in the TitleState.hx file, I'm using KE version 1.5.3?

**Reply by notsouguy** [reply]
search for the if statement that changes the state to OutdatedSubState and replace it by 	"FlxG.switchState(new MainMenuState());"

**Reply by ItzEazich001** [reply]
thx
